contrib/NeonVM/stack.h

```diff
@@ -1,6 +1,8 @@
 #ifndef __STACK_H
 #define __STACK_H
 
+#define INITIAL_CALLSTACK       10

```

Comment:
This sort of thing is an implementation detail and doesn't need to be in the header file. Put it in the .c file.

Reply:
Moved.